packages/app/utils/datetime.ts

```diff
@@ -1,3 +1,4 @@
+// from timestamp
```

Comment:
Why this comment? I think we can remove, or do we have something useful to comment?

Reply:
the idea was to split date functions that use `timestamp` as an arg from date function that use `date` as an arg.

maybe we don't need the comment or we can be more explicit.

Another idea is to split into other file and have `timestamp.ts` and `datetime.ts`.

related: https://github.com/SwaprHQ/stackly-ui/pull/26#discussion_r1252926446

What do you think might be better for all the suggestions?

Reply:
I think I can just remove the comments for now.

Reply:
Looks good to me!